src/pipeline/resolver.h

```diff
@@ -136,9 +136,10 @@ namespace librealsense
 
                 void open()
                 {
-                    for (auto && kvp : _dev_to_profiles) {
-                        auto&& sub = _results.at(kvp.first);
-                        sub->open(kvp.second);
+                    std::map<int, stream_profiles>::reverse_iterator it;
```

Comment:
If we go with this approach I will add an appropriate comment here

Reply:
How does this affect other streams, e.g. motion?

Reply:
You're using `rbegin()` so you can just use `auto` inside the `for` loop and remove this line, by the way.

> I will add an appropriate comment here
Please do.

Reply:
Good idea